engine/release-notes/24.0.md

```diff
@@ -19,6 +19,34 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 24.0.2
+
+{% include release-date.html date="2023-05-26" %}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 24.0.2 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A24.0.2)
+- [moby/moby, 24.0.2 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A24.0.2)
+
+### Bug fixes and enhancements
+
+* Fix a panic when building Dockerfiles that used locally built images. [moby/moby#45582](https://github.com/moby/moby/pull/45582)
+* Fix `docker build` hanging when running many concurrent builds. [moby/moby#45620](https://github.com/moby/moby/pull/45620)
+* Fix DNS resolution on Windows ignoring `etc/hosts` (`%WINDIR%\System32\Drivers\etc\hosts`),
+  including resolution of `localhost`. This was caused by a bug in Go's `netgo`
+  implementation, which was enabled in Go 1.19, but does not yet implement file-based
+  resolution. We disabled this feature in the Windows builds, now using the Windows
+  host resolver instead. [moby/moby#45562](https://github.com/moby/moby/pull/45562)
+* Apply a workaround for a containerd bug that causes concurrent `docker exec` commands to take significantly longer than expected. [moby/moby#45625](https://github.com/moby/moby/pull/45625)
+* containerd-storage: Fix an issue where the image `Created` field would contain an incorrect value. [moby/moby#45623](https://github.com/moby/moby/pull/45623)
+* containerd-storage: Adjust the output of image pull progress to match the output without the containerd-storage enabled. [moby/moby#45602](https://github.com/moby/moby/pull/45602)
+* containerd-storage: Prevent switching between containerd-storage and non-containerd-storage without restarting the daemon. [moby/moby#45616](https://github.com/moby/moby/pull/45616)
```

Comment:
It looks like we actually called the feature-flag `containerd-snapshotter` -- I think we should update all the release notes for uniformity/to match.

Reply:
I changed it, but then changed it back; I guess it's a bit of a combo. Maybe "snapshotter" is too much implementation specific, and the preferences mention "containerd image storage", so probably fine to leave it (for now)
<img width="947" alt="Screenshot 2023-05-26 at 10 45 00" src="https://github.com/docker/docs/assets/1804568/08d8a656-092e-4d68-891c-320ed9f45bae">

Reply:
IMO this should say `containerd image store:` instead of referring to the configuration flag or `containerd-storage`

Reply:
works for me; let me update

Reply:
Updated, and slightly rephrased the last bullet (the one for switching the feature)